site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
The test gallery still uses ts. Why don't we just use the same approach that use their for this file as well?